templates/partial/_featured-charms.html

```diff
@@ -1,29 +1,45 @@
 {% for charm in featured_charms %}
-<div class="l-flex__card u-equal-height" id="mysql">
+<div class="l-flex__card u-equal-height" id="{{ charm['name'] }}">
   <a href="/{{ charm['name'] }}" class="p-card--button">
     <div class="p-card__header">
       <div class="p-card__thumbnail-container">
-        <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,c_fill,w_64,h_64/{{ charm['icon'] }}" alt="{{ charm['name'] }} icon" class="p-card__thumbnail" width="40" height="40" loading="lazy">
+        {{
+          image(
+            url=charm.result.media[0].url,
```

Comment:
The icons are sometimes missing from the API. With the production API this page is showing a 500 error because of this. You can do something like we are doing here:
https://github.com/canonical-web-and-design/charmhub.io/blob/18bd1fe4191bf43e7dc012d4071f8c1e46d3d654/templates/details/_details-header.html#L14-L38